### ISSUE TITLE
AMP disallowed `!important` CSS qualifier

### ISSUE THREAD

**Issue by adamhenson** [TryGhost/Ghost#13941]
### Issue Summary

I received a notification from Google Search Console explaining that a problem with my page could cause it to not show up in search results.

> Errors can prevent your page or feature from appearing in Search results. The following errors were found on your site:

> Usage of the !important CSS qualifier is not allowed.

<img width="357" alt="Screen Shot 2022-01-09 at 10 18 02 PM" src="https://user-images.githubusercontent.com/747743/148715125-a1fe05e8-5f92-49de-aebf-c581c03f6851.png">

When viewing source of `/amp` pages... I do see the `!important` CSS qualifier used 4 times.

```css
.kg-header-card a.kg-header-card-button {
  display: flex;
 ...
  text-decoration: none !important;
 ...
}

...

.kg-header-card.kg-style-image a.kg-header-card-button,
.kg-header-card.kg-style-dark a.kg-header-card-button {
  ...
  color: #15171a !important;
}

.kg-header-card.kg-style-accent a.kg-header-card-button {
  ...
  color: var(--ghost-accent-color) !important;
}
```

I thought maybe I could fix this myself, so I downloaded the theme (Casper 4.7.2), but I couldn't find that style anywhere and also I didn't see the `amp.hbs` as described in [this post](https://ghost.org/docs/tutorials/custom-amp-themes/#how-does-amp-work-in-ghost).

### Steps to Reproduce

1: View source of this page for example: https://www.foo.software/posts/website-performance-in-2021-react-server-components-next-js-static-generation-and-linaria-zero-runtime-css/amp/

### Ghost Version

Latest (not sure - I use the "latest" tag from the Docker image)

### Node.js Version

Not sure - use the Docker Image

### How did you install Ghost?

Docker

### Database type

SQLite3

### Code of Conduct

- [X] I agree to be friendly and polite to people in this repository

**Comment by daniellockyer** [comment]
Hey @adamhenson! 👋🏻 This was [fixed](https://github.com/TryGhost/Ghost/commit/76cde7918687304af21b3f89c8ca51399f9d3bea) and released in 4.32.1 so please update using Ghost-CLI 🙂

**Comment by adamhenson** [comment]
Thanks @daniellockyer

**Comment by adamhenson** [comment]
@daniellockyer - do you know when Docker versions are published? It would be great to publish this one. Even though it's a patch - the negative impact is pretty severe.

I tried with the `latest` tag and it installed `4.32.0`.

I also tried to install directly with Ghost-CLI within the container and I got the error seen below.

```
ghost-admin@foo-blog-7cd4db9fb9-j8wbq:~$ ghost update 4.32.1
✔ Checking system Node.js version - found v14.18.2
ℹ Ensuring user is not logged in as ghost user [skipped]
ℹ Checking if logged in user is directory owner [skipped]
✔ Checking current folder permissions
✔ Checking memory availability
✔ Checking free space
✔ Checking for available migrations
✔ Checking for latest Ghost version
All up to date!
ghost-admin@foo-blog-7cd4db9fb9-j8wbq:~$ ghost --version    
Ghost-CLI version: 1.18.1
Ghost version: 4.32.1 (at ~)
ghost-admin@foo-blog-7cd4db9fb9-j8wbq:~$ ghost restart
Ghost instance is not running! Starting...
✖ Starting Ghost
A GhostError occurred.

Message: 
TypeError: Cannot read property 'connection' of undefined
    at Object.sanitizeDatabaseProperties (/home/ghost-admin/versions/4.32.1/core/shared/config/utils.js:62:25)
    at Object.loadNconf (/home/ghost-admin/versions/4.32.1/core/shared/config/loader.js:53:16)
    at Object.<anonymous> (/home/ghost-admin/versions/4.32.1/core/shared/config/index.js:3:25)
    at Module._compile (internal/modules/cjs/loader.js:1085:14)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1114:10)
    at Module.load (internal/modules/cjs/loader.js:950:32)
    at Function.Module._load (internal/modules/cjs/loader.js:790:12)
    at Module.require (internal/modules/cjs/loader.js:974:19)
    at require (internal/modules/cjs/helpers.js:93:18)
    at bootGhost (/home/ghost-admin/versions/4.32.1/core/boot.js:347:18)


Debug Information:
    OS: Debian GNU/Linux, v10
    Node Version: v14.18.2
    Ghost Version: 4.32.1
    Ghost-CLI Version: 1.18.1
    Environment: production
    Command: 'ghost restart'

Additional log info available in: /home/ghost-admin/.ghost/logs/ghost-cli-debug-2022-01-10T13_47_33_881Z.log

Try running ghost doctor to check your system for known issues.

You can always refer to https://ghost.org/docs/ghost-cli/ for troubleshooting.
ghost-admin@foo-blog-7cd4db9fb9-j8wbq:~$ ghost run
The `ghost run` command is used by the configured Ghost process manager and for debugging. If you're not running this to debug something, you should run `ghost start` instead.
TypeError: Cannot read property 'connection' of undefined
    at Object.sanitizeDatabaseProperties (/home/ghost-admin/versions/4.32.1/core/shared/config/utils.js:62:25)
    at Object.loadNconf (/home/ghost-admin/versions/4.32.1/core/shared/config/loader.js:53:16)
    at Object.<anonymous> (/home/ghost-admin/versions/4.32.1/core/shared/config/index.js:3:25)
    at Module._compile (internal/modules/cjs/loader.js:1085:14)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1114:10)
    at Module.load (internal/modules/cjs/loader.js:950:32)
    at Function.Module._load (internal/modules/cjs/loader.js:790:12)
    at Module.require (internal/modules/cjs/loader.js:974:19)
    at require (internal/modules/cjs/helpers.js:93:18)
    at bootGhost (/home/ghost-admin/versions/4.32.1/core/boot.js:347:18)
```

**Comment by daniellockyer** [comment]
@adamhenson The Docker images for Ghost are unofficial, but it seems they've already been published: https://github.com/docker-library/ghost/commit/a710352ab58e65982bc7e1abaa2ae06d5eb068bb

**Comment by adamhenson** [comment]
Awesome - thanks @daniellockyer! And sorry - didn't realize the Ghost Docker official image isn't truly official :D Yes, I noticed that in GitHub yesterday too, but DockerHub was only reflecting `4.32.0`. But, today - sure enough DockerHub is reflecting `4.32.1` 🙌 

I just had to recreate with the latest and all good! (`kubectl rollout restart ...`)

<img width="413" alt="Screen Shot 2022-01-11 at 10 16 48 AM" src="https://user-images.githubusercontent.com/747743/148971577-ed56ae4c-3abc-4329-8632-d30afd754f89.png">

Again, thanks for the quick response... and sorry this was a dupe.
